content-security-policy/reporting/report-cross-origin-no-cookies.sub.html

```diff
@@ -10,19 +10,25 @@
 </head>
 <body>
 <script>
-  var test = async_test("Image should not load");
-  fetch(
-    "/cookies/resources/set-cookie.py?name=cspViolationReportCookie1&path=" + encodeURIComponent("{{domains[www1]}}:{{ports[http][0]}}/"),
-    {mode: 'no-cors', credentials: 'include'})
-  .then(() => {
-    // This image will generate a CSP violation report.
-    const img = new Image();
-    img.onerror = test.step_func_done();
-    img.onload = test.unreached_func("Should not have loaded the image");
+  promise_test(function(test) {
```

Comment:
Any particular reason why this was changed to a promise test?
Not saying it's wrong just curious.